lib/options-handler.js

```diff
@@ -42,6 +47,8 @@ class ClientOptionsHandler {
         this.supportsExecute = Object.values(this.supportsExecutePerLanguage).some(value => value);
         const libs = this.parseLibraries(this.compilerProps(languages, 'libs'));
 
+        const cookieFileHash = ClientOptionsHandler.getFileHash(path.join('static', 'cookies.html'));
```

Comment:
This line had been giving me problems, I don't seem to be able to get it to work. The problem arises when the Amazon instances try to resolve this path, usually resulting in a enoent error. Any help would be great :)

Reply:
Hmm, right...I'll take a look and try and work out what's going on here. I think there's some hash versioning stuff going on :/

Reply:
also, this should be configurable like the PUG files; then they both should use the same file (at the moment it's only by convention that this file here is the same one that's actually linked to in the "Policies" dropdown)

Reply:
I'm still having some problems trying to get the paths to behave, so I've not been able to configure it in the most recent push :/

Reply:
No worries. We can merge and fix as we go anyway. I don't want this to be all on you, but I thought I'd leave the EU branch alone for now. My wife's proof-reading the privacy policy I wrote now...so...I might check that (And a cookie policy) into the EU branch, but I don't want to do anything else just yet :)